heron/executor/src/python/heron_executor.py

```diff
@@ -277,6 +280,7 @@ def parse_args(args):
     parser.add_argument("--topology-defn-file", required=True)
     parser.add_argument("--state-manager-connection", required=True)
     parser.add_argument("--state-manager-root", required=True)
+    parser.add_argument("--state-manager-config-file", required=True)
```

Comment:
config/yaml/heron.aurora needs update to add this param.

Reply:
:( required. Is it possible to be optional?

Reply:
I don't see any scheduler passes this flag. Why it is not causing errors?

Reply:
it is added in SchedulerUtils that every scheduler calls except aurora I believe

Reply:
I see. That could be the case.

Another note is that it seems all the other schedulers have statemanager configs explicitly in the following file. It may make sense to have this new config as well, especially since it is a required field.

heron/config/src/yaml/conf/kubernetes/statemgr.yaml
heron/config/src/yaml/conf/marathon/statemgr.yaml
heron/config/src/yaml/conf/yarn/statemgr.yaml
and so on.

Reply:
Never mind. I see what you mean. You want to override the config file itself.

Reply:
@jerrypeng  Cant add you as reviewer. :(  Here is the PR for aurora when you get a minute. I am testing the change internally.